### DISCUSSION TITLE
New Tab "Bookmarks"

### DISCUSSION THREAD

**Discussion by javathehunt** [bromite/bromite#1310] [Q&A]
When selecting the New Tab option a screen opens and there are 4 rows and 4 columns of bookmarks of sites I have been to. At first I thought this was a "frequently used" list of sites but I do not think that is the case at all. They are not bookmarks that I can tell. I can see how to remove them but can someone tell me what they are / how they got there and how can I specifically add certain shortcuts / bookmarks?

Thanks

**Comment by OmlineEditor** [comment]
I think we need to make a convenient button for adding a site.
I can't make a request for this function because only for security can I open a request.

![Screenshot_2](https://user-images.githubusercontent.com/13348933/178293112-3d38abe9-97b9-4efa-a743-2322ea55d668.jpg)
